ansible/roles/shared_dir.yml

```diff
@@ -4,6 +4,7 @@
   hosts: shared_dir_host  # Should be one
   become: yes
   vars_files:
+    - ../group_vars/all.yml
     - shared_dir/vars/main.yml
```

Comment:
Possible we could just move these into group_vars? I mostly just did this as mvp to fix the error. not sure about best practices.

Reply:
or we could just get rid of the shared dir completely, I did a quick look and it's not used in many places and almost everywhere it is used it's optional

Reply:
Merging this for now and adding it to the tech dept trello